OptionalParameter[] optional_parameters # List of supported and/or required optional parameters
AGVAction[] agv_actions                 # List of all actions with parameters supported by this AGV
